lib/features/children/overview/widgets/family_available_page.dart

```diff
@@ -14,15 +15,26 @@ class FamilyAvailablePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final currentUserFirstName = context.read<AuthCubit>().state.user.firstName;
+    // The Givt user profile is first in the list
+    final sortedAdultProfiles = profiles.where((p) => p.isAdult).toList()
```

Comment:
shall we rather incapsulate such logic (with getter?) somewhere in cubit/state level instead of widget/ui?

Reply:
great idea!